MIP/mip-21/README.md

```diff
@@ -0,0 +1,70 @@
+- **Authors**: [Primata](mailto:primata@movementlabs.xyz)
+
+## Abstract
+
+A bridge mechanism is essential for any Layer 2 solution, serving as the user's first interaction with the network. This process must be smooth and frictionless, ensuring that dealing with tokens on L1 and bridging to native gas tokens on L2 does not overcomplicate the user experience. The bridge must balance user onboarding, cost-efficiency, and simplicity, while minimizing transaction complexity. We propose a bridge based on attestors, inspired by the Cross-Chain Transfer Protocol (CCTP) by USDC, which offers an elegant solution to these challenges.
+
+## Motivation
+
+Our current bridge solution has been increasing in complexity with each iteration. Originally requiring three on-chain transactions, it now requires five. Additionally, the bridge does not currently support gas charging mechanisms to prevent exploits, and ongoing discussions are leaning toward adding further components, making it even more complex and computationally expensive.
+
+We believe the best approach is to draw from live implementations that have achieved a streamlined user experience with minimal cost to the user. One such example is USDC's [CCTP](https://www.circle.com/en/cross-chain-transfer-protocol), which offers simplicity, security, and efficiency in cross-chain bridging. Instead of increasing the complexity of our current bridge design, we should aim to adopt this proven mechanism.
+
+## Specification
+
+The Cross-Chain Transfer Protocol (CCTP) operates using a set of attestors that validate all bridge transactions occurring between chains. These attestors listen for bridge events, confirm the validity of transactions, and, if necessary, revert them by providing appropriate proofs. A transfer allowance managed by a centralized authority further ensures that breaches are mitigated by limiting the amount of value transferred.
+
+To align with this approach and reduce complexity:
+
+- **Bridge Relayer as Attestor**: The Bridge Relayer in our system will function as an attestor, operating via a multisig. This relayer, through aggregated signatures, produces a proof on a signed message.
+  
+- **User Interaction**: Users, or other parties interested in completing the bridge, retrieve their funds on the target chain by submitting the signature and message.
+
+- **Validation Mechanism**: The off-chain signature produced by the multisig is verified against the message on-chain. The challenge is to make sure that the signature is only used once, but the beauty is that the signature is comprised of a multi-party system that do not need to provide onchain signatures. But relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.
```

Comment:
```suggestion
- **Validation Mechanism**: The off-chain signature produced by the multisig scheme is verified against the message on-chain. 
The challenge is to make sure that the signature is only used once, but the beauty is that the signature is provided by a multi-party system that do not need to provide onchain signatures. 
Relying on a protocol like CCTP that has been tested in live environments significantly reduces the risk of any unexpected utilization of off-chain and on-chain proofs.
```

Reply:
Differentiated "multisig" (which is a signature but meant was the scheme) and "the signature".